lib/membrane_vpx/encoder/vp8_encoder.ex

```diff
@@ -0,0 +1,39 @@
+defmodule Membrane.VP8.Encoder do
+  @moduledoc """
+  Element that encodes a VP8 stream
+  """
+  use Membrane.Filter
+
+  alias Membrane.{VP8, VPx}
+
+  def_options encoding_deadline: [
+                spec: non_neg_integer(),
+                default: 1,
```

Comment:
Do you think this is a reasonable value? With regular streams it's much faster than realtime, which might affect quality.

Reply:
i went with the safest one, do you have an opinion what would be a good default?